app/src/main/res/layout/demo_activity.xml

```diff
@@ -433,12 +433,12 @@
             android:text="Above settings don't apply" />
 
         <Button
-            android:id="@+id/btn_show_hosted_login_age"
+            android:id="@+id/btn_show_universal_login"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginBottom="20dp"
-            android:text="Show Hosted Login Page" />
+            android:text="Show Login Page" />
```

Comment:
Why not Universal here ? 😄

Reply:
Because https://github.com/auth0/Lock.Android/pull/449#discussion_r160254544